### DISCUSSION TITLE
Curiousity about the settings this supports changing.

### DISCUSSION THREAD

**Discussion by CodingWithAnxiety** [T-Troll/alienfx-tools#321] [Ideas]
So in AWCC, I am able to edit the polling-rate delay on my AW610M. I'm curious if this is also possible in FX-tools?

Thanks in advance.

**Comment by T-Troll** [comment]
I only support light system for now.
It's interesting how they implement it (is it driver or mouse-related), but i don't have it for check.

**Reply by CodingWithAnxiety** [reply]
I believe it's mouse-related, though I cannot be sure. The issue I have is that when AWCC isn't open, the polling rate resets to default of 1 (or 1000 hz), which is over-the-top and causes MAJOR performance issues for me. 

To be perfectly frank- this is the only reason I can't really completely-switch to AFX tools, since I need to be able to decrease the polling rate. Though, if I can help you figure out how this feature works and implement it, I'd be more then happy to do so.

**Comment by CodingWithAnxiety** [comment]
![image](https://user-images.githubusercontent.com/78890450/227059687-af2c4fa6-1e43-4e28-b4e0-25e46b898977.png)


Image if this helps- There's a few more features in the AWCC program itself, as well;
![image](https://user-images.githubusercontent.com/78890450/227059778-58deda16-0df3-416b-b548-643b3eebcc27.png)
![image](https://user-images.githubusercontent.com/78890450/227059800-788d35e9-33c6-4b0f-bdd9-90fd47ea1836.png)
![image](https://user-images.githubusercontent.com/78890450/227059833-7e92ae9c-e7ec-4f8e-bfe7-ca57591dce52.png)

**Reply by T-Troll** [reply]
I check the situation.

Bad news - it's a custom filter driver, so DPI can't be touched.
Good news - for polling rate, one Russian guy write it's own [driver/GUI](https://github.com/LordOfMice/hidusbf/tree/master), which can change it. A bit tricky, but works (especially if you need to decrease rate, so you don't need to patch original Windows driver).

**Comment by JazzyJazzjazzy** [comment]
https://github.com/LordOfMice/hidusbf/blob/master/hidusbf.zip

Download.
Extract the contents of the ZIP file
Open the folder you just extracted.
Open the "DRIVER" folder.
Double-click the Setup.exe file.
Click your mouse in the list of connected devices.
Select your preferred polling rate using the drop-down menu next to "Selected Rate" at the bottom.
Click Install Service at the bottom.
Click Restart at the bottom.

**Reply by T-Troll** [reply]
I share him the link to same repo. But it's tricky to install (kernel driver!)
